Sources/TuistKit/Utils/XCResultParser.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import XCResultKit
+
+final class XCResultParser {
+    private let resultFile: XCResultFile
+    private let invocationRecord: ActionsInvocationRecord
```

Comment:
Let's remove these properties in favor of returning `XCResultFile` (and possibly `invocationRecord`) from the `parse` method